packages/mwp-app-render/src/components/uxcapture/UXCaptureConfig.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+
+type callbackType = (label: string) => mixed;
```

Comment:
probably `void` return type? The return value can't be consumed by anything, so better to let consumers know that they shouldn't return anything.